pyscriptjs/examples/webgl/raycaster/index.html

```diff
@@ -21,8 +21,8 @@
   </div>
 <script src='https://cdnjs.cloudflare.com/ajax/libs/three.js/89/three.min.js'></script>
 
-<script defer src="/build/pyscript.js"></script>
-<link rel="stylesheet" href="/build/pyscript.css" />
+<script defer src="../build/pyscript.js"></script>
```

Comment:
I think this is a typo... `js --> css`

Reply:
From `../build/pyscript.css` -> `href="../../build/pyscript.css"`

Reply:
I think this typo correction from Fabio is incorrect please change to:
`href="../../build/pyscript.js"`